memcache/memcache.php

```diff
@@ -435,19 +435,46 @@ function memcache_delete () {}
  */
 function memcache_debug ($on_off) {}
 
-function memcache_get_stats () {}
+/**
+ * @var object $memcache
+ */
+function memcache_get_stats ($memcache) {}
```

Comment:
looks like this method accepts more than one parameter http://php.net/manual/en/memcache.getstats.php